src/fork_gc.c

```diff
@@ -71,8 +71,8 @@ static void FGC_unlock(ForkGC *gc, RedisModuleCtx *ctx) {
 static RedisSearchCtx *FGC_getSctx(ForkGC *gc, RedisModuleCtx *ctx) {
   RedisSearchCtx *sctx = NULL;
   if (gc->type == FGC_TYPE_NOKEYSPACE) {
-    sctx = rm_malloc(sizeof(*sctx));
-    *sctx = SEARCH_CTX_STATIC(ctx, gc->sp);
+    // TEMPORARY
+    sctx = NewSearchCtxFromSpec(ctx, gc->sp);
```

Comment:
So this is still not a safe access right? Will be handled on another PR?

Reply:
Yes. #3336